python/tvm/meta_schedule/testing/tlcbench.py

```diff
@@ -0,0 +1,90 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=invalid-name,import-outside-toplevel
+# type: ignore
+"""Model loader for TLCBench."""
+import os
+import logging
+import tvm
+from tvm import relay
+from tvm.error import TVMError
+from tvm.contrib.download import download_testdata
+
+
+log = logging.getLogger("tlcbench")
+
+
+def convert_to_qnn(onnx_path, json_path, params_path, batch_size, seq_len):
+    """Run the ONNX frontend and the FQ2I pass. The output is serialized to disk."""
+    import onnx
+
+    onnx_model = onnx.load(onnx_path)
+
+    shape_dict = {
```

Comment:
I don't think so, in pytorch case the names of inputs don't matter, but for onnx they need to match the one in the *.onnx file.

Reply:
but yeah, the current interface doesn't make it obvious what the input names are, which are needed for deployment. I can add a comment describing what the input names are.

Reply:
or we can return `shape_dict` to the caller, that way the caller doesn't need to hard-code input names / shapes. What do you think?

Reply:
Oh really? I thought I needed to match that all the time. 
I see. If that's the case, it looks good to me. 
Maybe it would be better to fix `relay_workload.py` side for the consistency. 

Thank you!

Reply:
I mean, the names of inputs you pass to the PT frontend and the ones you use in `runtime.set_input(input_name, ...`)` must match, off course. But for the PT the actual value of `input_name` can be whatever, while for onnx we need to use exactly the same name recorded in the input onnx model.

Reply:
Oh somehow your later comments were not shown when I was leaving the comment. 
Yeah, personally, I would prefer returning `shape_dict` to the caller cause I'm a lazy person :)

Reply:
Sure, check out this commit https://github.com/apache/tvm/pull/10622/commits/752b812bdd83172cf9e9c26171aaacf7cba92c30

Reply:
Cool. LGTM. Thanks!